src/Client/AcsfClientFactory.php

```diff
@@ -45,4 +47,17 @@ public function fromCredentials(string $username, string $api_key, string $site_
     return new AcsfClient($username, $api_key, $this->logger, ['base_uri' => $site_name . '/api/v1/']);
```

Comment:
I think it would make sense to instantiate the client in the factory and pass it to the acsf client as a parameter. What do you think?

Reply:
Updated code in https://github.com/acquia/acsf-contenthub-console/pull/19/commits/9662768f52540486e2830f369229e1b41d250d9e